third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+
+realdds is a library to help control a realsense camera over network using librealsense API.
+It is a work in progress aimed at, ultimately, enabling remote users to do with the camera everything a user can do with a USB connected camera, and more (multiple clients for frames, etc...)
+
+realdds uses DDS topics to send and receive information to/from the camera. See https://github.com/IntelRealSense/librealsense/blob/dds/doc/DDS/DDS%20ICD.md
```

Comment:
It would be more pretty if the link was not explicit - I mean if writing: "See DDS ICD", while the DDS ICD is a link to this url.

Reply:
Fixed

Reply:
ok